libs/coin-framework/src/derivation.ts

```diff
@@ -168,14 +168,20 @@ const modes = Object.freeze({
   polkadotbip44: {
     overridesDerivation: "44'/354'/<account>'/0'/<address>'",
   },
-  gliflegacy: {
+  glifLegacy: {
     overridesDerivation: "44'/1'/0'/0/<account>",
     tag: "legacy",
+    mandatoryEmptyAccountSkip: 5,
   },
   glif: {
-    overridesDerivation: "44'/461'/0'/0/<account>",
+    overridesDerivation: "44'/<coin_type>'/<address>'/<node>/<account>",
+    mandatoryEmptyAccountSkip: 5,
+  },
+  filecoinBIP44: {
+    overridesDerivation: "44'/<coin_type>'/<account>'/<node>/<address>",
     startsAt: 1,
     tag: "third-party",
```

Comment:
Just wondering if this `"third-party"` tag still makes sense? Is it clear for Live users?

Reply:
I would change this one to another tag, but if we remove it, it will break current LL which are using this tag. Do you think there is a better approach?

Reply:
I have applied some changes on tags. What about now?